python/python/glide/async_commands/cluster_commands.py

```diff
@@ -415,6 +415,41 @@ async def function_delete(
             ),
         )
 
+    async def fcall_ro_route(
+        self,
+        function: str,
+        arguments: Optional[List[str]] = None,
+        route: Optional[Route] = None,
+    ) -> Optional[TClusterResponse[TResult]]:
```

Comment:
Oh wait `TResult` includes `None` too, so you can revert it.
```suggestion
    ) -> TClusterResponse[TResult]:
```
Anyhow, the `TClusterResponse` isn't optional.

Reply:
Update below too